### PR TITLE
Improve rotary scrolling

### PR DESCRIPTION
**Pull request by saryongkang**
- fixed issue that position indicator is not viewed in PersonListScreen
- fixed list item not to be cropped by position indicator or bottom edge
- moved focusRequest to Modifier extension and guarded by SideEffect function (thanks @yschimke)

* I think FocusRequester is not necessary because it's special one for focusing each Composable not entire ComposeView.

### CODE REVIEW

File: wearApp/src/main/java/com/surrus/peopleinspace/RotaryEventHandler.kt [hunk 1]
```diff
@@ -12,18 +13,24 @@ import androidx.compose.ui.composed
...
 
 @OptIn(ExperimentalComposeUiApi::class)
-fun Modifier.scrollHandler(scrollState: ScrollableState): Modifier = composed {
+fun Modifier.rotaryEventHandler(scrollState: ScrollableState): Modifier = composed {
     val context = LocalContext.current
     val scope = rememberCoroutineScope()
     val scaledVerticalScrollFactor =
-        remember { ViewConfiguration.get(context).getScaledVerticalScrollFactor() }
+        remember { ViewConfiguration.get(context).scaledVerticalScrollFactor }
+    val view = LocalView.current
```
Comment (yschimke) [inline wearApp/src/main/java/com/surrus/peopleinspace/RotaryEventHandler.kt:27]: Nice improvements.

**Comment by yschimke** [comment]
Looks good.  My one concern with view.requestFocus() in either the existing code or this, is an assumption that only a single item on screen is scrollable.  Wouldn't work if you wanted a date picker with different elements scrollable.

But nice cleanup for this usage.

**Review by yschimke — APPROVED** [review]

**Comment by saryongkang** [comment]
@yschimke Oh, that's good point. I assumed only one scrollable control will be needed in each screen. Let me think more about it.

**Comment by yschimke** [comment]
I wouldn't delay this, it's cleaner and functionally equivalent.

**Comment by saryongkang** [comment]
Oh, thanks. I've found that `FocusManager` or `FocusRequester` doesn't call `View.requestFocus()`. I will stay with current commit. ;)

**Pull request merged**